pinot-core/src/main/java/org/apache/pinot/core/common/MinionConstants.java

```diff
@@ -58,8 +60,27 @@ private MinionConstants() {
   }
 
   public static class MergeRollupTask {
-    public static final String TASK_TYPE = "mergeRollupTask";
-    public static final String MERGE_TYPE_KEY = "mergeTypeKey";
+    public static final String TASK_TYPE = "MergeRollupTask";
+
+    public static final String COLLECTOR_TYPE_KEY = "collectorType";
+
+    // Rollup aggregate function related configs
+    public static final String AGGREGATE_KEY_PREFIX = "aggregate";
+
+    // Merge properties related configs
+    public static final String MERGE_KEY_PREFIX = "merge";
+    public enum Granularity {
```

Comment:
I think that we should put `Granularity` somewhere else. We can put this somewhere together with `MergeRollupTaskMetadata`

Reply:
Moved this enum into `MergeRollupTaskMetadata`